blog/2023-09-08-newsletter.md

```diff
@@ -0,0 +1,185 @@
+---
+title: "Conda Newsletter, Issue #2"
+slug: "2023-09-08-newsletter"
+description: |
+    Upcoming Events, conda-store brief, Is conda free?, pre and post plugin hooks, Recent releases, Travis's latest talk on Youtube.
```

Comment:
```suggestion
    Upcoming Events, conda-store brief, Is conda free?, pre and post plugin hooks, Recent releases, Travis' latest talk on Youtube.
```

With the name "Travis", the possessive just ends with a single `'` and not `'s`; Trust me on this 😉.